articles/batch/batch-low-pri-vms.md

```diff
@@ -18,12 +18,27 @@ The tradeoff for using low-priority VMs is that those VMs may not always be avai
 
```

Comment:
We should try to make this generic so we can somewhat align with spot branding: "Use lower cost compute with Batch" ?

This entire paragraph/section should probably be aligned as well, as well as the entire doc. I'd prefer not to use "low priority" as the generic term for preemptible-class VMs.

Reply:
As discussed at the last Spot meeting, we will use "Spot" as the catch-all generic term throughout. Resolving.